src/cls/rbd/cls_rbd.cc

```diff
@@ -4625,6 +4626,10 @@ static const std::string REMOTE_STATUS_GLOBAL_KEY_PREFIX("remote_status_global_"
 static const std::string INSTANCE_KEY_PREFIX("instance_");
 static const std::string MIRROR_IMAGE_MAP_KEY_PREFIX("image_map_");
 static const std::string REMOTE_NAMESPACE("remote_namespace");
+static const std::string GROUP_KEY_PREFIX("group_");
+static const std::string GROUP_GLOBAL_KEY_PREFIX("gglobal_");
+static const std::string GROUP_STATUS_GLOBAL_KEY_PREFIX("gstatus_global_");
+static const std::string GROUP_REMOTE_STATUS_GLOBAL_KEY_PREFIX("gremote_status_global_");
```

Comment:
This will cause issues with the group listing as the filter "group_" will match all the key types.

How about `status_group_global` and `remote_status_group_global` etc.